src/test/java/picoded/dstack/struct/simple/StructSimple_FileWorkspaceMap_test.java

```diff
@@ -202,6 +202,48 @@ public void fileWrite_andFolderMove() {
 		assertFalse(fileWorkspace.fileExist("test/folder/file.txt"));
 		assertTrue(fileWorkspace.fileExist("moved/folder/file.txt"));
 	}
+
+	//-----------------------------------------------------------------------------------
+	//
+	// Copy test
+	//
+	//-----------------------------------------------------------------------------------
+	
+	@Test
+	public void fileWrite_andFileCopy() {
+		// Get the file workspace to use
+		FileWorkspace fileWorkspace = testObj.newEntry();
+		assertNotNull(fileWorkspace);
+		
+		// Write file
+		fileWorkspace.writeString("test/folder/file.txt", "anything");
+		assertTrue(fileWorkspace.fileExist("test/folder/file.txt"));
+		
+		// Move it
+		fileWorkspace.copyFile("test/folder/file.txt", "test/folder/moved.txt");
+		
+		// File moved
+		// assertFalse(fileWorkspace.fileExist("test/folder/file.txt"));
```

Comment:
assertTrue, to check that the original file is there

we should also improve the test case to check the file content